public/app/plugins/datasource/grafana-azure-monitor-datasource/query_ctrl.ts

```diff
@@ -100,6 +106,9 @@ export class AzureMonitorQueryCtrl extends QueryCtrl {
           ? this.datasource.azureLogAnalyticsDatasource.defaultOrFirstWorkspace
           : '',
     },
+    azureResourceGraph: {
+      resultFormat: 'table',
```

Comment:
Actually, I think it might be better to hold back on something like that and think about what that default query should be.

I agree there should be some kind of prompt to get users started, but I'm not entirely happy with how it is for Log Analytics at the moment. I don't like how that is a syntactically invalid query that generates errors on first load :(

Happy for this to stay empty for the time being while we have a think about it.